client/signup/steps/site-type/form.jsx

```diff
@@ -51,29 +58,24 @@ class SiteTypeForm extends Component {
 							onClick={ this.handleSubmit.bind( this, siteTypeProperties.slug ) }
 						>
 							<strong className="site-type__option-label">{ siteTypeProperties.label }</strong>
-							<span className="site-type__option-description">
-								{ siteTypeProperties.description }
-							</span>
-							{ ! this.props.isJetpack && siteTypeProperties.purchaseRequired && (
+							{ showDescriptions && (
+								<span className="site-type__option-description">
+									{ siteTypeProperties.description }
+								</span>
+							) }
+							{ showPurchaseRequired && siteTypeProperties.purchaseRequired && (
 								<Badge className="site-type__option-badge" type="info">
-									{ this.props.translate( 'Purchase required' ) }
+									{ translate( 'Purchase required' ) }
 								</Badge>
 							) }
 						</Card>
 					) ) }
 				</Card>
-			</Fragment>
+			</>
 		);
 	}
 }
 
-export default connect(
-	state => ( {
-		// TODO: Better handling for Jetpack flows in the site-type lib,
-		// so we don't depend on injecting conditionals into components like this.
-		isJetpack: !! isJetpackSite( state, getSelectedSiteId( state ) ),
-	} ),
-	{
-		recordTracksEvent,
-	}
-)( localize( SiteTypeForm ) );
+export default connect( {
```

Comment:
This is breaking the site type step because of an invalid `mapStateToProps` param.

```suggestion
export default connect( null, {
```

Reply:
Thanks for catching that!